src/apps/Organization/Apps.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, GU, textStyle } from '@aragon/ui'
+import LocalIdentityBadge from '../../components/IdentityBadge/LocalIdentityBadge'
+import { AppType } from '../../prop-types'
+import Label from './Label'
+
+const Apps = ({ appsLoading, apps, shortAddresses }) => {
+  const apmApps = apps.filter(app => !app.isAragonOsInternalApp)
```

Comment:
For the other components here, we do this sort of filtering in `Organization.js`. I think it makes sense to keep it that way, and to pass these child components the minimum number of things they need.